src/excalidraw-utils/webpack.prod.config.js

```diff
@@ -0,0 +1,40 @@
+const webpack = require("webpack");
+const path = require("path");
+
+module.exports = {
+  mode: "production",
+  resolve: {
+    extensions: [".tsx", ".ts", ".js"],
+  },
+  optimization: {
+    runtimeChunk: false,
+  },
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    filename: "excalidraw-utils.js",
+    library: "ExcalidrawUtils",
+    libraryTarget: "umd",
+  },
+  entry: "./index.js",
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx|js|jsx|mjs)$/,
```

Comment:
```suggestion
        test: /\.(ts|js)$/,
```

Reply:
See above.
I was able to remove `jsx` and `mjs` but we have a "dependency" on at least one `tsx` file.